packages/api/src/auth/decoders/azureActiveDirectory.ts

```diff
@@ -0,0 +1,49 @@
+import jwt from 'jsonwebtoken'
+import jwksClient from 'jwks-rsa'
+
+export const verifyAzureActiveDirectoryToken = (
+  bearerToken: string
+): Promise<null | Record<string, unknown>> => {
+  return new Promise((resolve, reject) => {
+    const { AZURE_ACTIVE_DIRECTORY_AUTHORITY } = process.env
+    if (!AZURE_ACTIVE_DIRECTORY_AUTHORITY) {
+      throw new Error(
+        '`AZURE_ACTIVE_DIRECTORY_AUTHORITY` env vars are not set.'
```

Comment:
```suggestion
        '`AZURE_ACTIVE_DIRECTORY_AUTHORITY` env var is not set.'
```

Reply:
Fixed in https://github.com/jeliasson/redwood/commit/a222300bc5282a2fabef17362ff9eefdd572f1b8